### PR TITLE
feat(components/login/passwordReset): add stepper to component

### PR DESCRIPTION
**Pull request by davidmartin84**

### CODE REVIEW

File: components/login/passwordReset/src/index.scss [hunk 1]
```diff
@@ -57,11 +57,24 @@ $bgc-login-password-reset-item: #ffffff; // TODO: Delete
...
+
+    &Title {
+      letter-spacing: -1px;
+      margin: 0;
+    }
+
+    &Message {
+      color: $c-login-password-reset-main-text;
+      margin: $m-l 0 0;
+      line-height: $lh-login-password-reset-message;
```
Comment (AgonisticKatai) [inline components/login/passwordReset/src/index.scss:73]: Could we alphabetize these properties?

**Review by oegea — APPROVED** [review]
🦸‍♀️

**Review by AgonisticKatai — APPROVED** [review]

**Review by AgonisticKatai — COMMENTED** [review]
It would be interesting in the future to add a test to verify that it meets the desired specifications.

**Review by jordevo — APPROVED** [review]

**Comment by davidmartin84** [comment]
@AgonisticKatai We have a task to do a 18n functionality test

**Pull request merged**